lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -77,13 +77,21 @@ def authorize(money, payment, options = {})
         commit_transaction('authorize', post)
       end
 
-      def capture(money, authorization, _options = {})
+      def capture(money, authorization, options = {})
         post = {
           transaction: { id: authorization }
         }
-        post[:order] = { amount: amount(money).to_f } if money
+        verify_flow = options[:type] && options[:value]
+
+        if verify_flow
+          add_customer_data(post, options)
+          add_verify_value(post, options)
+        elsif money
```

Comment:
`elsif` is added here instead of `else` so that the capture can work in case of **nil** amount as well.